cli/cmd/migrate_v3.go

```diff
@@ -101,6 +101,10 @@ func migrateConnectionV3(ctx context.Context, sourceClient *managedplugin.Client
 				return handleSendError(err, writeClients[i], "migrate")
 			}
 		}
+
+		if err := migrateSummaryTable(writeClients[i], destinationTransformers[i], destinationSpecs[i]); err != nil {
```

Comment:
Shouldn't we check the `send_sync_summary` option is enabled first?

Reply:
Because `send_sync_summary` is on each destination we would have to iterate through all of the destinations. The first part of the for loop where we iterate through the destination we do check if it is enabled though: https://github.com/cloudquery/cloudquery/blob/97babd42ea9edb27b874d3a30a51cd5e11bdf7e5/cli/cmd/summary.go#L122-L124

Reply:
> Because `send_sync_summary` is on each destination we would have to iterate through all of the destinations. The first part of the for loop where we iterate through the destination we do check if it is enabled though:

In this file (`cli/cmd/migrate_v3.go`) we don't call `send_sync_summary`, but call `migrateSummaryTable` directly so I don't see where we check the config option.
You can test by running `cloudquery migrate` with the following config:

```yaml
kind: source
spec:
  name: "gcp"
  path: "cloudquery/gcp"
  registry: "cloudquery"
  version: "v12.7.0"
  tables: ["gcp_storage_buckets"]
  spec:
    project_ids: ["cq-playground"]
  destinations: ["postgresql"]
---
kind: destination
spec:
  name: "postgresql"
  path: "cloudquery/postgresql"
  version: "v8.0.4"
  spec:
    connection_string: "postgresql://postgres:pass@localhost:5432/postgres?sslmode=disable"
```

And see that the summary table is created even when the option is disabled

Reply:
Nice catch fixed in [4fdcd88](https://github.com/cloudquery/cloudquery/pull/17842/commits/4fdcd8881f685c4123983adca14f344da77babc3)